spec/features/manage_users_spec.rb

```diff
@@ -74,4 +74,36 @@
       expect( User.last.first_name ).to eq @user.first_name
     end
   end
+
+  describe "inactivating users" do
+    before :each do
+      @user = create :user
+      visit root_path
+      within ".admin_country_select" do
+        select @user.country.name
+        click_on "Update"
+      end
+    end
+
+    it "has users active by default" do
+      select @user.name, from: :edit_user_id
+      click_on "Edit User"
+
+      expect( @user.active ).to be true
+    end
+
+    it "confirms before marking as inactive" do
+      select @user.name, from: :edit_user_id
+      click_on "Edit User"
+      delete_link = find_link 'Delete User'
+      
+      expect(delete_link['data-confirm']).to eq "You are about to inactivate this user. Are you sure?"
```

Comment:
Probably want to expand this to say "and when I click on the button, I can see that the user _has_ been deactivated" (meaning that they don't show up)